portfolio/src/main/webapp/style.css

```diff
@@ -73,3 +73,16 @@ ol{
 label{
   margin-bottom: 0;
 }
+
+.homepage{
```

Comment:
nit: I can't find the explicit rule for this, but I think to stay consistent with other whitespace rules you should put a space between the class/id name and the opening brace, like in the style guide - https://google.github.io/styleguide/htmlcssguide.html#Declaration_Stops - same elsewhere.

Reply:
Done!